open_gateway/app.py

```diff
@@ -678,12 +696,37 @@ def main():
                 app.config["MODEL_JSON"] = json.load(open(arg))
             else:
                 print("Model json file was not found!")
+        elif opt in ("-i", "--classmap_images_json_path"):
+            print(f"{C_CLR_OKBLUE}Start saving classmap images:")
+            if os.path.exists(arg):
+
+                images_read = json.load(open(arg))
+                app.config["CLASS_MAP_IMAGES"] = []
+                dir_to_save  = os.path.join(app.static_folder, CLASS_MAP_IMG_FLD_NAME)
+
+                image_manager = ImageManager(dir_to_save=dir_to_save)
+
+                for class_name, img_path in images_read.items():
```

Comment:
should img_path be relative to the class_map json or relative to the open gateway application?

it might make more sense to have it relative to the class_map json

Reply:
agree, relative to the json is most logical